### DISCUSSION TITLE
a blue line between badges in readme overview ??

### DISCUSSION THREAD

**Discussion by ezhillragesh** [community/community#79043] [Accessibility] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/84183052/72c609e0-4e05-4a26-ae36-09537921e447)
Can anyone say why is this happening and I a m using a tag and adding my badge with img tag. Thanks

**Comment by ghostinhershell** [comment]
Hey there @ezhillragesh, 

Thanks so much for reaching out in our community. Can you send me the URL to the page where you're seeing this so that I can take a closer look?

**Comment by ezhillragesh** [comment]
https://github.com/SourceGuideHQ @ghostinhershell  please check this out :)

**Comment by ezhillragesh** [comment]
@ghostinhershell  any updates ?

**Reply by leeringrose** [reply]
@ezhillragesh I think you haven't given <a> tag's `text-decoration` property to `none` on CSS.
Generally I recommend you to <a> link elements' `text-decoration` to `none` overall across your project.
![image](https://github.com/community/community/assets/132311826/a8499a47-bf91-4595-80bc-647a487e9ca3)
![image](https://github.com/community/community/assets/132311826/3ca993c9-54f6-4eb1-b8fa-c446d62f0955)

**Reply by leeringrose** [reply]
Oh.. sorry I didn't aware it's markdown written. sorry for my confusion. 🤣

I think it's related to the online-source where you are using those badge images from. Can you please try with other resources to fetch images from.

**Reply by ezhillragesh** [reply]
yeah sure let me try with other badge source

**Comment by jindalujjwal0720** [comment]
**It's the problem with the anchor tag text-decoration underline css**, I tried turning it off but the github sanitises all the css.
_To GitHub:_ I think at least CSS can be turned on in markdown, it will help designing markdown alot and I don't think basic css will bring much security issues.

**Comment by milinddhamu** [comment, marked as answer]
If you correctly format the <a> tag with <img> tag , the blue line will not appear.
Wrong -> `<a> 
<img/>
</a>
`
rather try this -> `<a><img/></a>`

it will surely work.
Main issue raises in formatting and indenting the tags but its Markdown so write in one-line.

**Reply by jermanuts** [reply]
>rather try this -> <a><img/></a>

Thanks a lot, it worked. The problem was that there is spaces between tags, remove them!

**Reply by ezhillragesh** [reply]
yeah sure

**Reply by ezhillragesh** [reply]
changed

**Reply by Lfbyte** [reply]
Thanks!!!
